### PR TITLE
Implemented code for static UI of product inventory detail screen (#1wn3fmm)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: src/views/ProductInventoryDetail.vue [hunk 1]
```diff
@@ -0,0 +1,779 @@
...
+
+.variant,
+.actions {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.text-center {
```
Comment (dt2patel) [inline src/views/ProductInventoryDetail.vue]: is there a better alternative here. like some kind of selector?

**Review by dt2patel — CHANGES_REQUESTED** [review]
Take a look at suggestion.

File: src/views/ProductInventoryDetail.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/ProductInventoryDetail.vue]: I have used selector here sir for aligning text of all ion-label in `open-orders` class but if I do so it is aligning text of first row labels also which we dont want that is why I have explicitly added it to desired labels.

Comment (dt2patel) [reply src/views/ProductInventoryDetail.vue]: what about a not first child selector?

Comment (dt2patel) [reply src/views/ProductInventoryDetail.vue]: or maybe just add another selector that selects first child and resets text align property

Comment (azkyakhan) [reply src/views/ProductInventoryDetail.vue]: Yes I am exploring it sir

Comment (azkyakhan) [reply src/views/ProductInventoryDetail.vue]: Used the not first child selector.

localhost_8100_product-inventory-detail

File: src/views/ProductInventoryDetail.vue [hunk 2]
```diff
@@ -0,0 +1,779 @@
...
+
+.variant,
+.actions {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.open-orders > ion-card > ion-item > ion-label:not(:first-child)  {
```
Comment (dt2patel) [inline src/views/ProductInventoryDetail.vue:756]: I think we can make this just for ion-item and select not first child and not last child.

**Review by dt2patel — CHANGES_REQUESTED** [review]
I think we can take different approach in how we select the label.

File: src/views/ProductInventoryDetail.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/ProductInventoryDetail.vue:756]: Do we want our last child also to be aligned at start ?
Do we only want middle child to aligned center ?

Comment (dt2patel) [reply src/views/ProductInventoryDetail.vue:756]: now that selector is updated, please remove classes from above

**Review by dt2patel — CHANGES_REQUESTED** [review]
One last change and then we can merge this.

**Comment by azkyakhan** [comment]
> One last change and then we can merge this.

Done sir

**Review by dt2patel — APPROVED** [review]

**Pull request merged**